libs/langchain/langchain/vectorstores/surrealdb.py

```diff
@@ -0,0 +1,434 @@
+import asyncio
+from typing import (
+    Any,
+    Iterable,
+    List,
+    Optional,
+    Tuple,
+)
+
+from surrealdb import Surreal
```

Comment:
this should not be imported globablly (after doing the below, it will cause errors)

rather, it should be imported inside __init__ of this class (wrapped in a nice try-except to catch errors)

Reply:
This is now done.